miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/MiniappSdkInitializer.kt

```diff
@@ -68,6 +75,7 @@ class MiniappSdkInitializer : ContentProvider() {
             context = context,
             miniAppSdkConfig = MiniAppSdkConfig(
                 baseUrl = manifestConfig.baseUrl(),
+                rasProjectId = manifestConfig.rasProjectId(),
```

Comment:
`com.rakuten.tech.mobile.ras.AppId` has not been removed yet (deprecated) so we can add the backward support here. If manifestConfig.rasProjectId() is empty then

```rasProjectId = manifestConfig.rasAppId()```

Reply:
Yep, @minh-rakuten is right. See my comment on the "preview mode" PR. Since the value is a boolean, it can't be empty or null, but I think this should work:
```
rasProjectId = manifestConfig.rasProjectId() || manifestConfig.rasAppId()
```

Reply:
Thanks, I have updated this part.